File: php/URI/1134.php
<?php

$cliente = 0;
$alcool = 0;
$gasolina = 0; 
$diesel = 0;
         
while ($cliente != 4) {

    $cliente = trim(fgets(STDIN));
    switch ($cliente) {
        case 1:
            $alcool++;
            break;
        case 2:
            $gasolina++;
            break;
        case 3:
            $diesel++;
            break;
        case 4:
            $cliente = 4;
            break;
    }
}

echo "MUITO OBRIGADO\n";
echo "Alcool: " . $alcool . "\n";
echo "Gasolina: " . $gasolina . "\n";
echo "Diesel: " . $diesel . "\n";

?>

File: C/URI/1134.c
#include <stdio.h>
 
int main() {
 
int gasolina, alcool, diesel, opcao;
        gasolina = 0;
        alcool = 0;
        diesel = 0;
        do{
          scanf("%d", &opcao);
                if(opcao == 2)
                        gasolina = gasolina + 1;
                else if(opcao == 1)
                        alcool = alcool + 1;
                else if(opcao == 3)
                        diesel = diesel + 1;
        }       while (opcao != 4);

        printf("MUITO OBRIGADO\n");
        printf("Alcool: %d\n", alcool);
        printf("Gasolina: %d\n", gasolina);
        printf("Diesel: %d\n", diesel);


 
    return 0;
}